### ISSUE TITLE
[Feature] 垂直桑基图如何设置节点的水平位置呢？How to custom node's horizontal position on a vertical sankey?

### ISSUE THREAD

**Issue by lutian197** [apache/echarts#20449]
### What problem does this feature solve?

如何实现垂直桑基图的两侧节点靠边？nodeGap只能统一设置

### What does the proposed API look like?

--

**Comment by gmhhope** [comment]
Yes, it is annoying that the nodeGap could not set for individual columns but only could be set uniformly across each column which create something informative like the below one:

<img width="800" alt="Screenshot 2024-11-29 at 2 48 42 PM" src="https://github.com/user-attachments/assets/9b002e7a-d5c2-4450-b832-3f15863012cc">

Instead if we can do it on inidividual column or automatically set to identify proper node gaps for each column, it will be very likely be more useful.

<img width="800" alt="Screenshot 2024-11-29 at 2 48 29 PM" src="https://github.com/user-attachments/assets/379fb63b-ba0d-444d-8fb8-1573db9c0a1d">

**Comment by lutian197** [comment]
@gmhhope

**Comment by lutian197** [comment]
@gmhhope 
Thank you. I will think about it
